reference-page/app/templates/route-components/assets.hbs

```diff
@@ -165,7 +181,13 @@
 <section>
 	<h2 id="stores">Mobile apps stores</h2>
 	<p>
-		Asset names for mobile apps stores are defined by the pattern <code><strong>wds-company-store-STORE_NAME</strong></code>,
+		To help with the promotion of our community apps Google Play and App Store badges should be displayed
+		when possible. Each badge should be translated to our supported languages and point to the respective
```

Comment:
We don't support translations ([ticket](https://wikia-inc.atlassian.net/browse/XW-1919)). Should it be here, then?